docs/assembly-control-unit.md

```diff
@@ -171,47 +281,105 @@ Heat  |  Stir |
     Cool    Light
 ```
 
-Note that in our assembly, the cables inside the Control Unit are color-coded per circuit (blue for heat, red for cool, green for stir, and black for the light). When we assemble the cable later, we will simply use the ribbons in their regular order without any color coding by function.
+<IMG ALT="Cable order" SRC="./media/control-unit/connector.png" WIDTH="300" />
+
+<br>
+<br>
 
 <IMG ALT="8-pin relay sub-assembly parts" SRC="./media/control-unit/8-pin-relay-sub-assembly-parts.jpg" WIDTH="600" />
 
-<IMG ALT="8-pin relay sub-assembly assembled" SRC="./media/control-unit/8-pin-relay-sub-assembly.jpg" WIDTH="600" />
 
-Note that in the image above, 3 (+) and 3 (-) wires are shown for connecting to the relay board even though the 4x board only requires one of each (+/-).
 
 **Components:**
 
-- 8-pin female Phoenix connector (the part with screw terminals that the other part with the screw terminals fit into)
-- 3x individual relays (or 4x relay board)
-- 2x 2-wide lever nuts (or another way to join 4 small wires)
-- 2x 3-wide lever nuts (or another way to join 6 small wires)
-- Assorted wires
-  - 4x red wires and 4x black wires (~4" for 5v relay power - only one of each is needed for a 4x relay board)
-  - 3x short wires (~4" for connecting the relays to the 8-pin Phoenix connector - preferably color-coded)
-  - 2x 3-wire and 1x 4-wire cables (~6" for connecting the relays to the +12v power and the 8-pin Phoenix connector to the -12v power)
-- 3x male-to-female leads (for signaling from Pi to relays)
-- USB cable (cut off with the "A" side intact and the + / - wires exposed and stripped)
+| Section | Part ID      | Part Name     | Count | Notes |
+|--- | ------------- | ------------- |------| ----|  
+|||
+8Pin Relay|SP-PC8|Phoenix Connectors 8-pin - female side|2
+8Pin Relay|CU-RLY|4 Channel Relay Board - 5V|1
+8Pin Relay|CU-WCL|Wire connectors|
+8Pin Relay|CU-WCL|Wire connectors|
+8Pin Relay|SP-CAB|Cables (Control to Pumps)|15ft| Used to make short wires
+8Pin Relay|CU-JMP|Breadboard Jumper Wires - male-female|5
+|||
+
+
+
+
+
+**Prepare Wires**
+
+- Take the 4-wire cable (SP-CAB) and cut two 4" sections and three 6" sections.
+   - Take one of the 4" sections and separate out the red and black wires to use
+   - Take the second 4" section and sepereate all four colors (red, blue, green, black)
+   - Take one of the 6" sections and remove the black wire leaving a red, blue, andgreen 3-wire cable to use
```

Comment:
Let's make them all 4-wire for now. This is related to the other comment about removing mention of the light and button.

Reply:
Should we have them strip all the colors in the cable or tell them when to skip a wire when they won't use all 4 colors? Won't we need single red and black wires for the wire connectors?